pkg/convert/go.go

```diff
@@ -0,0 +1,69 @@
+package convert
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/message/pipeline"
+)
+
+func ToGo(m model.Messages) ([]byte, error) {
+	pipelineMsg := messagesToPipeline(m)
+
+	msg, err := json.Marshal(pipelineMsg)
+	if err != nil {
+		return nil, fmt.Errorf("encode pipeline.Messages to JSON: %w", err)
+	}
+
+	return msg, nil
+}
+
+func FromGo(b []byte) (model.Messages, error) {
+	var pipelineMsg pipeline.Messages
+
+	err := json.Unmarshal(b, &pipelineMsg)
+	if err != nil {
+		return model.Messages{}, fmt.Errorf("decode JSON to pipeline.Messages: %w", err)
+	}
+
+	msg := messagesFromPipeline(pipelineMsg)
+
+	return msg, nil
+}
+
+func messagesToPipeline(m model.Messages) pipeline.Messages {
```

Comment:
do we really need, two additonal functions for that ? e.g. messagesToPipeline and messagesFromPipeline, imo it decreases readability as namings for all functions are too similar to each other and that way makes it more confusing

Reply:
I'm not sure that it is good idea because if we will combine these two functions then first of all the function length will be too long which isn't good practice according to clean code book ;D. The second we will mix two functionalities which is also not great.